src/screens/SearchScreen.tsx

```diff
@@ -24,21 +36,40 @@ const styles = StyleSheet.create( {
 } )
 
 const SearchScreen = () => {
-  const [ , setSearch ] = useState( '' )
+  const [ searchValue, setSearch ] = useState( '' )
   const [ , setPageCount ] = useState( 0 )
+  const searchInputRef = useRef<TextInput>( null )
 
   const handleTextChange = ( v: string ) => {
     setSearch( v )
     setPageCount( 0 )
   }
-
+  const clearInput = () => {
+    searchInputRef.current?.clear()
+    setSearch( '' )
+  }
   return (
     <Container statusBarColor={Colours.MediumGray}>
       <View style={styles.searchStrip}>
-        <View style={styles.searchBar}>
-          <SearchBar onChangeText={handleTextChange} />
+
+        <BackButton label={<AntIcon name="arrowleft" size={24} />} style={styles.backButton} />
+
+        <View style={styles.searchInput}>
+
+          <View style={styles.searchBar}>
+            <SearchBar ref={searchInputRef} onChangeText={handleTextChange} />
+          </View>
+
+          {OS.android && ( searchValue.length > 1 ) && (
```

Comment:
Out of interest, why is this needed?

Also, the block underneath should be indented one level in.

Reply:
The default search component on iOS includes a clear button for dirty input, apparently Android does not and needs to be added manually.

Reply:
```suggestion
          {OS.android && searchValue.length && (
```

@saihaj surely you mean `>0`, and not `>1`?